libvast/builtins/formats/json.cpp

```diff
@@ -454,6 +437,16 @@ auto make_parser_impl(generator<chunk_ptr> json_chunk_generator,
       continue;
     }
     for (auto doc_it = stream.begin(); doc_it != stream.end(); ++doc_it) {
+      // doc.error() will inherit all errors from *doc_it and get_value.
+      // No need to check after each operation.
+      auto doc = (*doc_it).get_value();
+      if (auto err = doc.error()) {
+        ctrl.warn(caf::make_error(
+          ec::parse_error, fmt::format("failed to fully parse '{}' : {}. "
+                                       "Some events can be skipped.",
```

Comment:
```suggestion
          ec::parse_error, fmt::format("skips invalid JSON '{}' : {}",
```

Reply:
What about this?

```suggestion
          ec::parse_error, fmt::format("failed to parse JSON and may skip some events '{}' : {}",
```

The reason being that the printed JSON in the log message could be rather lengthy—so I think we should print the outcome first.

Also, stylig-wise, please use a single sentence for warnings that starts lower-case and has no dot at the end, otherwise it's inconsistent with the remaining warnings.